app/assets/javascripts/govuk_publishing_components/analytics/pii.js

```diff
@@ -12,6 +12,9 @@
   var UNLOCK_TOKEN_PATTERN = /unlock_token=[a-zA-Z0-9-]+/g
   var STATE_PATTERN = /state=.[^&]+/g
 
+  // specific URL parameters to be redacted from smart answer URLs
+  var VACCINATION_STATUS_PATTERN = /vaccination_status=.[^&]+/g
```

Comment:
I'm not very regex knowledgeable but I wanted to understand what this does so tried it out on regxr.com. Is it possible that this matches too much if the `vaccination_status` field is present but blank? I don't know if that would ever be possible though.

![Screenshot 2022-01-26 at 16 03 18](https://user-images.githubusercontent.com/861310/151200018-86aaaa9b-d513-4a73-993a-9e53a6bf3797.png)